.github/workflows/e2e.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Install and build npm dependencies
         run: npm ci
       - name: Install Chromium
-        run: echo "chromium_binary=$(npx @puppeteer/browsers install chromium@latest | cut -f 2- -d' ')" >> $GITHUB_ENV
+        run: echo "chromium_binary=$(npx @puppeteer/browsers install chromium@1141358 | cut -f 2- -d' ')" >> $GITHUB_ENV
```

Comment:
let's add a file (e.g., `.browser`) to the root of the repository containing `chromium@1141358` and read that file in CI to get the pinned browser@version?